src/lib/orion_websocket/parser.cpp

```diff
@@ -30,10 +30,19 @@
 #include "rapidjson/writer.h"
 
 #include "logMsg/logMsg.h"
+#include "rest/HttpHeaders.h"
 
-void ws_parser_parse(std::string &url, std::string &verb, std::string &payload, const char *msg)
+
+#include <map>
```

Comment:
Style guide ...

Include order:
  1. C include files
  2. C++ include files
  3. Third party software
  4. Our own headers (starting with headers from other libs and then the headers for *this* lib)

Not my idea, but I kind of agree ...